README.md

```diff
@@ -152,6 +152,55 @@ using (IDbConnection conn = new SnowflakeDbConnection())
 }
 ```
 
+<a id="sample-connection-strings"></a>
+#### Using Special Characters in Connection Passwords
+
+The .NET connector now uses Microsoft [DbConnectionStringBuilder](https://learn.microsoft.com/en-us/dotnet/api/system.data.oledb.oledbconnection.connectionstring?view=dotnet-plat-ext-6.0#remarks) to following .NET specification for escaping characters in connection strings. 
```

Comment:
Does "now" mean from a specific version number onward?
If so, should we just include that version number? (e.g. "For version n and more recent versions")

Reply:
Beginning with version 2.0.18, the .NET connector uses Microsoft